frontend/bank-transfer-ui/src/app/main/transfer-list/transfer-list.component.ts

```diff
@@ -1,15 +1,93 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {TransferService} from '../../core/transfer/transfer.service';
+import {Transfer} from '../../core/transfer/transfer.model';
+import {ActivatedRoute} from '@angular/router';
+import {ModalDismissReasons, NgbModal} from '@ng-bootstrap/ng-bootstrap';
+import {NgxSpinnerService} from 'ngx-spinner';
 
 @Component({
   selector: 'app-transfer-list',
   templateUrl: './transfer-list.component.html',
   styleUrls: ['./transfer-list.component.css']
 })
 export class TransferListComponent implements OnInit {
+  transfer: Transfer[];
+  batch: number;
```

Comment:
Te 'wszystkie' propertisy powinny być Twoim modelem

Reply:
Czyli stworzyć model do Listy transferow i wrzucić tam pola ?

Reply:
Listra transferów (Transfer[]) jest ok, ale poniżej masz dużo property np. związana z sortowaniem. Jeśli w innej części aplikacji też będziesz miał sortowanie, to byś musiał kopiować wszystkie property, a jak stworzysz sobie model do tego, to będziesz go reużywał wszędzie